src/app/zap-templates/helper-chip.js

```diff
@@ -69,6 +69,12 @@ function isCommandAvailable(clusterSide, incoming, outgoing, source, name) {
   return false;
 }
 
+function isString(type) { return type == 'CHAR_STRING' || type == 'OCTET_STRING'; }
+
+function isLongString(type) { return type == 'LONG_CHAR_STRING' || type == 'LONG_OCTET_STRING'; }
+
+function isNullable(type, isArray) { return isString(type) || isLongString(type) || isArray; }
```

Comment:
So default for strings is null, not "zero-length string"?  That might end up changing....

Reply:
I think this is `NULL` because the string is not part of the bytes of the payload. By spec it is not supposed to be here.
I have not looked deeply but I imagine that "zero-length string" is potentially a valid string for some APIs.